index.html

```diff
@@ -8,241 +8,321 @@
     <h1>Pokedex</h1>
     <div class="grid-container">
         <div class="pokemon"><img src="images/1.png"><h2>Bulbasaur</h2><h3>#001</h3><div class="type grass">
-            <div class="tooltip"><span class="tooltiptext">Grass</span></div>
-        </div><div class="type poison">        
-            <div class="tooltip"><span class="tooltiptext">Poison</span></div>
+            <div class="tooltip"><span class="tooltiptext">Grass</span></div>Grass
```

Comment:
As you've added the text for each of the types, you could get rid of the tool tips I had made as theres not much point in them now.

Reply:
I think I have done this and the commit appears to appear in the pull request?

Reply:
Perfect, I can see you've removed it 😄